### ISSUE TITLE
Download FCE XML data programmatically

### ISSUE THREAD

**Issue by tomshen** [ScottyLabs/course-api#11]
As discussed in #10. Possibly integrate [cmu-grades's auth module](https://github.com/willcrichton/cmu-grades/blob/master/auth.py).

**Comment by justingallagher** [comment]
This is implemented and seems to work, follow #18 for bug fixing progress.

**Comment by justingallagher** [comment]
Reopening this issue as the feature has been removed (since we couldn't get it working again after #27). It would be nice to have it back though!
